src/main/java/org/students/simplebitcoinnode/repository/AdjacentNodeRepository.java

```diff
@@ -1,18 +1,8 @@
 package org.students.simplebitcoinnode.repository;
 
 import org.springframework.data.jpa.repository.JpaRepository;
-import org.springframework.data.jpa.repository.Query;
 import org.students.simplebitcoinnode.entity.AdjacentNode;
 
-import java.util.List;
-
-public interface AdjacentNodeRepository extends JpaRepository<AdjacentNode, Long> {
-    /**
-     * Finds best nodes by hash rate
-     * @param maximumHashRate specifies the maximum hash rate one node can have
-     * @param limit specifies the query limit
-     * @return a list containing AdjacentNode objects
-     */
```

Comment:
Why is `findBestNodesByHashRate` deleted by your changes? That method is necessary for the miner to work properly.